apps/backend/src/libs/modules/encrypt/encrypt.ts

```diff
@@ -0,0 +1,6 @@
+import { BaseEncrypt } from "./base-encrypt.module.js";
+
+const encrypt = new BaseEncrypt();
+
+export { encrypt };
+export { type Encrypt } from "./types/types.js";
```

Comment:
Lets inline this export to make it export { encrypt, type Encrypt } so we will have import statements in the beginning of the file and export statements in the end of the file. It will be convenient

Reply:
I tried to combine the export of encrypt and Encrypt as suggested, but the linter flagged an error: ESLint: Use export…from to re-export Encrypt.(unicorn/prefer-export-from). I had to export them separately to resolve the issue. Let me know if there's a better approach.

Reply:
Okey, then. Issue resolved